backend/internal/controller/images.go

```diff
@@ -0,0 +1,17 @@
+package controller
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/hoka-isdl/MERS/backend/internal/repository"
+	"github.com/hoka-isdl/MERS/backend/internal/schema"
+)
+
+func GetImagesHandlerFunc(ctx *gin.Context) {
+	var imageslist []schema.ListImagesInner
```

Comment:
[キャメルケース](https://e-words.jp/w/%E3%82%AD%E3%83%A3%E3%83%A1%E3%83%AB%E3%82%B1%E3%83%BC%E3%82%B9.html)で統一しよう！他の箇所もあればよろしく。
```suggestion
	var imagesList []schema.ListImagesInner
```